src/shared/util.js

```diff
@@ -1030,6 +1030,32 @@ function createObjectURL(data, contentType = "", forceDataSchema = false) {
   return buffer;
 }
 
+function getURL(str) {
```

Comment:
Also, since this function is only being used in the `src/core/` folder, please move it to `src/core/core_utils.js` instead to avoid (needlessly) duplicating this code in both the `pdf.js` and `pdf.worker.js` files.